src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -287,12 +289,21 @@ export default (domain, domains, global) => {
         serverConfig.push(['# handle .php', '']);
 
         const loc = `location ~ ${domain.routing.legacyPhpRouting.computed ? '[^/]\\.php(/|$)' : '\\.php$'}`;
+
+        const fastcgiPass = {
+            fastcgi_pass: domains.some(d => d.php.php.computed) && domain.php.phpBackupServer.computed !== ''
+                ? phpUpstream(domain) : phpPath(domain),
```

Comment:
```suggestion
            fastcgi_pass: domain.php.phpBackupServer.computed !== '' ? phpUpstream(domain) : phpPath(domain),
```
This is now site-specific, so we don't need to check other domains.